lib/loga/log_subscribers/event_wrappers/action_mailer_event.rb

```diff
@@ -0,0 +1,48 @@
+require 'active_support/core_ext/array/conversions'
+
+module Loga
+  module LogSubscribers
+    module EventWrappers
+      class ActionMailerEvent
+        EVENT_TYPE = 'action_mailer'.freeze
+
+        def initialize(event, *payload_fields)
+          @event          = event
+          @payload_fields = payload_fields
+        end
+
+        def to_loga_event
+          Event.new(data: data, message: to_s, type: EVENT_TYPE)
+        end
+
+        def to_s
+          "#{message} in #{event.duration}ms"
+        end
+
+        private
+
+        attr_reader :event, :payload_fields
+
+        def message
+          raw_message = exposed_payload.to_a.map do |tuple|
+            tuple.to_sentence(two_words_connector: ' is ')
```

Comment:
This looks weird to me:
```
action is hello_world and mailer is WelcomeMailer in 0.0004ms
```

Could we just use the messages from rails with a little edit:

e.g.

```
# def deliver
INFO: "#{mailer}##{action}: Sent mail to #{recipients} (#{event.duration.round(1)}ms)"
debug { event.payload[:mail] }

# def process
DEBUG: "#{mailer}##{action}: Processed outbound mail in (#{event.duration.round(1)}ms)"

# def receive
INFO "Received mail #{from} (#{event.duration.round(1)}ms)"
debug { event.payload[:mail] }
```

Also this class complects things or at least the way it is written. A simple method in `Loga::LogSubscribers::ActionMailer` should do it 🤷‍♂️